features/steps/flaw.py

```diff
@@ -106,3 +109,37 @@ def step_impl(context):
     assert len(rows) > context.flaws_count, \
         "No more flaws loaded after click the 'Load More Flaws' button"
     context.browser.quit()
+
+
+@when("I add a public comment to the flaw")
+def step_impl(context):
+    # Click the "Add public comment" button
+    context.browser.execute_script("window.scrollTo(0, document.body.scrollHeight);")
+    time.sleep(2)
+    wait_for_visibility_by_locator(context.browser, By.XPATH, COMMENT_BUTTON)
+    try:
+        element=context.browser.find_element(By.XPATH, COMMENT_BUTTON)
+    except NoSuchElementException:
+        context.browser.quit()
+    ActionChains(context.browser).move_to_element(element).click().perform()
+    # Add the random public comment
+    comment_text_windown = f"(//textarea[@class='form-control'])[5]"
+    wait_for_visibility_by_locator(context.browser, By.XPATH, comment_text_windown)
+    public_comment = generate_random_text()
+    context.browser.find_element(By.XPATH,comment_text_xpath).send_keys(public_comment)
```

Comment:
Can not find the define of the variable `comment_text_xpath`

Reply:
Updated.